docs/userguide/entry_point.rst

```diff
@@ -160,6 +160,9 @@ Then, we can add a GUI script entry point:
         [project.gui-scripts]
         hello-world = "timmins:hello_world"
 
+.. note::
+   To be able to import `PySimpleGUI`, you need to add `pysimplegui` to your package dependencies.
```

Comment:
```suggestion
   To be able to import ``PySimpleGUI``, you need to add ``pysimplegui`` to your package dependencies.
```

Yeah, I forgot the double backticks... This constant change between Markdown (used by Github) and RST used by sphinx...